### PR TITLE
Add productName label to CRs

### PR DESCRIPTION
**Pull request by austincunningham**
## Overview
The https://github.com/integr8ly/cloud-resource-operator/blob/master/pkg/resources/util.go functions creates CRO CR from information taken from Integreatly-operator. This PR adds
```yaml
metadata:
  label:
    productName: name
```
to the CR

Jira: https://issues.redhat.com/browse/INTLY-4377

### Steps to test locally

- clone this repo
- checkout this branch
- go to the go.mod in your local copy of integreatly-operator and add the following line to the bottom of the file
```replace github.com/integr8ly/cloud-resource-operator v0.0.0-20191125111132-250f22a81882 => /the local directory for your clone of the cloud resource operator```
- remove integreatly-operators existing vendor for cloud resources e.g.
```
vendor/github.com/integr8ly/cloud-resouces-operator
```
- make code/gen
- run the integreatly-operator locally as normal
### Verify
that there is 
```yaml
metadata:
  labels:
   productName: productNameHere
```
for types postgres, redis,blobstorage and smtpcredentialset in the integreatly namespace.

### CODE REVIEW

File: deploy/olm-catalog/cloud-resources/0.5.0/cloud-resources.v0.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,63 +8,75 @@ metadata:
...
-            "type": "workshop"
+            "type": "REPLACE_ME"
           }
         }
       ]
     capabilities: Basic Install
     categories: Integration & Delivery
     certified: "false"
-    containerImage: quay.io/integreatly/cloud-resource-operator:0.5.0
+    containerImage: quay.io/integreatly/cloud-resource-operator:0.4.0
```
Comment (dimitraz) [inline deploy/olm-catalog/cloud-resources/0.5.0/cloud-resources.v0.5.0.clusterserviceversion.yaml]: Can you change this to `0.5.0`?

Comment (austincunningham) [reply deploy/olm-catalog/cloud-resources/0.5.0/cloud-resources.v0.5.0.clusterserviceversion.yaml]: sure

**Pull request merged**